packages/core/src/types/external/OrgSchema/CreativeWork.ts

```diff
@@ -71,13 +71,29 @@ export type CreativeWork = Thing & {
   usageInfo?: CreativeWork | undefined;
 };
 
+type Person = {
+  '@type': 'Person';
+  description?: string | undefined;
+  image?: string | undefined;
+  name?: string | undefined;
+};
+
+const person = <TEntries extends ObjectEntries>(entries?: TEntries | undefined) =>
+  object({
+    description: orgSchemaProperty(string()),
+    image: orgSchemaProperty(string()),
+    name: orgSchemaProperty(string()),
+
+    ...entries
+  });
+
 export const creativeWork = <TEntries extends ObjectEntries>(entries?: TEntries | undefined) =>
   thing({
     // For forward compatibility, we did not enforce @type must be "CreativeWork" or any other subtypes.
     // In future, if Schema.org introduced a new subtype of CreativeWork, we should still able to parse that one as a CreativeWork.
 
     abstract: orgSchemaProperty(string()),
-    author: orgSchemaProperties(string()),
+    author: orgSchemaProperty(union([lazy(() => person()), string()])),
```

Comment:
Maybe we don't need `lazy()` as we don't have recursive yet.

Reply:
```suggestion
    author: orgSchemaProperty(union([person(), string()])),
```